pylearn2/train_extensions/tests/test_monitor_based_save_best.py

```diff
@@ -74,5 +75,24 @@ def test_tagging():
         model.monitor.channels['foobar'].val_record.append(3.0)
         ext.on_monitor(model, None, None)
         assert model.tag['test123']['best_cost'] == 3.0
+
+        # setting the starting epoch of saving models.
+        ext = MonitorBasedSaveBest(channel_name='foobar', tag_key='test12',
+                                   start_epoch=10, save_path=fn)
+        ext.setup(model, None, None)
+        assert model.tag['test12']['best_cost'] == float("inf")
+        # Best cost after one iteration.
+        model.monitor.channels['foobar'].val_record.append(5.0)
+        ext.on_monitor(model, None, None)
+        assert model.tag['test12']['best_cost'] == float("inf")
+        # Best cost after a second, better iteration.
+        model.monitor.channels['foobar'].val_record.append(3.0)
+        ext.on_monitor(model, None, None)
+        assert model.tag['test12']['best_cost'] == float("inf")
+        # Best cost after a third, worse iteration.
+        model.monitor.channels['foobar'].val_record.append(7.0)
+        ext.on_monitor(model, None, None)
+        assert model.tag['test12']['best_cost'] == float("inf")
+
```

Comment:
Could you explain the test logic please? I don't see where it's being tested after 10?